ScoreSaber/UI/Elements/Leaderboard/MapInfoView.cs

```diff
@@ -79,18 +80,16 @@ internal string GetMapStatusString() {
                 return $"Qualified - (<size=75%><color=#a6a6a6>{(_currentMapInfo.qualifiedDate.HasValue ? _currentMapInfo.qualifiedDate.Value.ToString("dd/MM/yy") : string.Empty)}</color></size>)";
             else if (isLoved)
                 return $"Loved - (<size=75%><color=#a6a6a6>{(_currentMapInfo.lovedDate.HasValue ? _currentMapInfo.lovedDate.Value.ToString("dd/MM/yy") : string.Empty)}</color></size>)";
-            else if (isUnranked)
-                return $"Unranked";
             else
-                return string.Empty;
+                return $"Unranked";
```

Comment:
should also delete the `isUnranked` variable now that it is unused

Reply:
fixed